pandas/tests/frame/test_operators.py

```diff
@@ -451,6 +451,18 @@ def test_arith_flex_frame(self):
         with tm.assert_raises_regex(NotImplementedError, 'fill_value'):
             self.frame.add(self.frame.iloc[0], axis='index', fill_value=3)
 
+    def test_arith_flex_zero_len_raises(self):
```

Comment:
shouldn't these be in test_arithmetic?

Reply:
Eventually yes.  The more important grouping for now is that this belongs with the test above it, which I plan to split/parametrize before moving to test_arithmetic